### DISCUSSION TITLE
IndentationError: unindent does not match any outer indentation level

### DISCUSSION THREAD

**Discussion by danteistanto** [teamalgoritma/community#372] [Q&A] [answered]
<img width="960" alt="error pd outer identation level" src="https://user-images.githubusercontent.com/112854600/193379192-004cca60-6b6f-453e-999d-455c645ad769.png">
 Mohon bantuannya, terimakasih sebelumnya

**Comment by kevbow** [comment]
Dear @danteistanto,

Coba cek kembali apakah ada spasi atau tab berlebih pada line 71 seperti yang tertera pada error. 
Penjelasan lebih lengkap mengenai error ini dapat diakses pada [link ini](https://stackoverflow.com/questions/26720841/python-indentationerror-unindent-does-not-match-any-outer-indentation-level)

Semoga membantu.

**Reply by dinaraulliyah** [reply]
Saya juga sama kak masalahnya di indentation, sudah coba hapus spaces yang di awal syntax tapi masih juga error

**Comment by onlyphantom** [comment, marked as answer]
Hello @dinaraulliyah  and @danteistanto , if you look closely you see that red squiggy line (berwarna merah) on line 70 and 73. What it tells you is that you have an extra space in every line of the document. 

Sila hapuskan spasi ekstra di setiap baris, karena spasi itu hanya dipakai untuk code indentation. Cari semua squiggy line yang berwarna merah -- itu tanda error karena syntax nya salah. 

If you want a shortcut, using VSCode, hit `CTRL` + `SHIFT` + `p` to bring out the command palette and then `Format Document`, it you don't have a formatter it will ask you to pick one and perform the installation. Proceed to install and then do `Format Document` again. Good luck!

**Reply by dinaraulliyah** [reply]
Ok, thank you!

**Reply by danteistanto** [reply]
ok, thank you

**Reply by danteistanto** [reply]
selamat pagi,
saya mengalami kendala yang sama di checkpoint 4, mohon bantuannya
<img width="960" alt="error checkpoint 4" src="https://user-images.githubusercontent.com/112854600/193485149-51895da8-a21c-412c-84e4-04e819188f86.png">

**Reply by danteistanto** [reply]
ketika saya lakukan Format Document, muncul sebagai berikut:
<img width="960" alt="notification format document" src="https://user-images.githubusercontent.com/112854600/193485665-03b8953d-3cd3-48da-9cd6-964aefd8c8a1.png">

**Reply by yosiaazarya** [reply]
Mungkin boleh dicoba bapak, karena seharusnya tidak bermsalah untuk indentation dari code yang kami berikan.

**Reply by yosiaazarya** [reply]
Mohon dipastikan bahwa setiap indentasi yang dibuat menggunakan "tab" dan jangan menggunakan "spasi" untuk membuat indentasi, karena hal tersebut akan sangat berpengaruh. Terima kasih.
